src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -92,15 +92,10 @@ public MvcRazorHost(ICodeTreeCache codeTreeCache)
                     ScopeManagerBeginMethodName = nameof(TagHelperScopeManager.Begin),
                     ScopeManagerEndMethodName = nameof(TagHelperScopeManager.End),
 
-                    OutputGenerateStartTagMethodName = nameof(TagHelperOutput.GenerateStartTag),
-                    OutputGenerateContentMethodName = nameof(TagHelperOutput.GenerateContent),
-                    OutputGenerateEndTagMethodName = nameof(TagHelperOutput.GenerateEndTag),
-
                     // Can't use nameof because RazorPage is not accessible here.
                     CreateTagHelperMethodName = "CreateTagHelper",
                     StartTagHelperWritingScopeMethodName = "StartTagHelperWritingScope",
                     EndTagHelperWritingScopeMethodName = "EndTagHelperWritingScope",
-                    HtmlEncoderPropertyName = "HtmlEncoder",
```

Comment:
Provide the `WriteTagHelper`/`WriteTagHelperTo` strings here. We don't want to depend on the default values.